demo/api/src/footer/entities/footer.entity.ts

```diff
@@ -1,30 +1,33 @@
-import { BlockDataInterface, RootBlock, RootBlockEntity } from "@comet/blocks-api";
-import { CrudSingleGenerator, RootBlockDataScalar, RootBlockType } from "@comet/cms-api";
+import { ExtractBlockData, RootBlock, RootBlockEntity } from "@comet/blocks-api";
+import { CrudSingleGenerator, DocumentInterface, RootBlockDataScalar, RootBlockType } from "@comet/cms-api";
 import { BaseEntity, Embedded, Entity, OptionalProps, PrimaryKey, Property } from "@mikro-orm/core";
 import { Field, ID, ObjectType } from "@nestjs/graphql";
-import { v4 as uuid } from "uuid";
+import { v4 } from "uuid";
 
 import { FooterContentBlock } from "../blocks/footer-content.block";
-import { FooterContentScope } from "./footer-content-scope.entity";
+import { FooterScope } from "../dto/footer-scope";
 
 @Entity()
-@ObjectType()
+@ObjectType({
+    implements: () => [DocumentInterface],
+})
```

Comment:
The document interface was intentionally removed in https://github.com/vivid-planet/comet/pull/1906. We should remove it in Starter as well.